fluentd/configs.d/input-ocp-audit.conf

```diff
@@ -0,0 +1,44 @@
+<source>
+  @type tail
+  path /var/log/audit/audit-ocp.log
+  pos_file /var/log/audit/audit-ocp.pos
+  # From 3.9, the advanced audits logs comes in json format by default.
+  format json
+  tag audit-ocp.log
+</source>
+
+<match audit-ocp.log>
+  # @type stdout
+   @type copy
+  <store>
+    @type elasticsearch_dynamic
+    log_level debug
+    host "#{ENV['OPS_HOST']}"
+    port "#{ENV['OPS_PORT']}"
+    scheme https
+    index_name operations.ocp.auditlog.
```

Comment:
@richm maybe we should take this as an opportunity to separate some of our log traffic in order not to bloat the indices.  I can see us putting this in some 'infra' index and aliasing it to 'operations' or  maybe it becomes a particular document type in 'operations' and we need to take advantage of rollover because of the volume this will add.

Reply:
> @richm maybe we should take this as an opportunity to separate some of our log traffic in order not to bloat the indices. I can see us putting this in some 'infra' index and aliasing it to 'operations' 

not sure what you mean

> maybe it becomes a particular document type in 'operations' 

Afaik types will be going away in Elasticsearch soon - deprecated in 5.x and unsupported in 6.x - so we'll need to use some mechanism other than types

Also - what about multi-tenancy and access control?  changes to the openshift-elasticsearch-plugin?  What about curator?

In general, I think this PR is premature and needs an actual design doc.